### PR TITLE
Add Java Spring Boot

### PR DESCRIPTION
**Pull request by romain325**
#48  Add Support for java spring boot

**DRAFT: Experienced problem with already existing theme (rely on uppercased key and return lowercased)**

#### Checklist

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/developer-authentication-demos/blob/master/contributing.md) before creating this PR.
- [x] This is not a duplicate entry for any framework or language.

##### Description of change

Add Java Spring Boot demo for Login Radius

### CODE REVIEW

**Comment by romain325** [comment]
Description of the issue described:

Quoting the code from [profile.js](https://github.com/LoginRadius/developer-authentication-demos/blob/master/theme/assets/scripts/profile.js):

```js
utility.setInnerText('profile_email', response.data.Email[0].Value);
```

The key is uppercased,
 but the the identity given by getProfileByAccessToken in [this file](https://github.com/LoginRadius/developer-authentication-demos/blob/c657da5bb953a6e24f5d07a7648268521bce8a2a/demos/java/spring/src/main/java/com/loginradius/springdemo/LoginRadiusServices.java) has lower cased key:

```java
authApi.getProfileByAccessToken(accessToken, null, new AsyncHandler<>() {
     @Override
     public void onSuccess(Identity identity) {
         response[0] = new Response("success","Profile Fetched", identity);
     }
     [...]
}
```

Does anybody have an idea how to work with this ?

**Comment by ashish8947** [comment]
@romain325 
You can refer to our [Documentation](https://www.loginradius.com/docs/developer/sdk-libraries/java-library/) and [Demo](https://github.com/LoginRadius/login-page-demos/tree/master/java-idx-demo) for more information

**Comment by ashish8947** [comment]
@romain325 
Please Add "Congigiure Demo " and "Running the Demo" section in your Readme file with Step by Step Process and it is good if you can add "prerequisites" too. For example you can checkout [this](https://github.com/LoginRadius/developer-authentication-demos/blob/master/demos/nodejs/express/readme.md) and please verify demo once on your local too.

**Comment by ashish8947** [comment]
@romain325 
Thanks for your contribution. I have tried to run your project but it is not in working condition. I can't see even an index page so please fix the same.

**Review by ashish8947 — CHANGES_REQUESTED** [review]
Request changes

**Comment by romain325** [comment]
@ashish8947 
I'm not able to reproduce your problem can you please bring me more information ?
There is no need of an index.html because we precise the path of the theme folder on the application.properties as a static resource location

File: demos/java/spring/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+spring.resources.static-locations=file:<YOUR-PATH>/developer-authentication-demos/theme
```
Comment (ashish8947) [inline demos/java/spring/src/main/resources/application.properties:1]: I replaced the URL below and it works. So please change the same here and it is ready to merge.
 
```
file:///<YOUR-PATH>/developer-authentication-demos/theme
```
Thanks @romain325

**Review by ashish8947 — CHANGES_REQUESTED** [review]

**Review by ashish8947 — APPROVED** [review]

**Pull request merged**